microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/arquillian/extension/ArquillianConfigurationExtension.java

```diff
@@ -0,0 +1,19 @@
+package org.jboss.eap.qe.microprofile.openapi.arquillian.extension;
+
+import org.jboss.arquillian.container.test.spi.RemoteLoadableExtension;
+import org.jboss.arquillian.core.spi.LoadableExtension;
+import org.jboss.eap.qe.microprofile.openapi.OpenApiTestCase;
+
+/**
+ * Extension which sets property descriptor in {@link OpenApiTestCase} class which is used in MP OpenAPI spec tests
+ */
+public class ArquillianConfigurationExtension implements RemoteLoadableExtension {
+
+    /**
+     * @param builder Instance of builder to be used by the registered extension
+     */
+    @Override
+    public void register(LoadableExtension.ExtensionBuilder builder) {
+        builder.observer(OpenApiTestCase.class);
```

Comment:
@fabiobrz I think that `ConfigUtils` should be used here as it's holder of configuration. `OpenApiTestCase` class can be removed.

Reply:
Done.